guides/common/modules/proc_enabling-capsule-in-UI.adoc

```diff
@@ -0,0 +1,55 @@
+[id="assigning-organization-location-capsule-server_{context}"]
+
+= Enabling the {SmartProxyServer} in the {Project} web UI
+
+After installing the {SmartProxyServer} packages, you must enable the {SmartProxyServer} in the {Project} web UI by assigning the correct organization and location to it.
+
+.Procedure
+
+. Log into the {Project} web UI.
+. Click *Organization*, then select *Any Organization*.
+. Click *Location*, then select *Any Location*.
+. Click *Hosts* > *All Hosts* and select the checkbox next to the {SmartProxyServer}. 
+. Click the *Select Actions* list and select *Assign Organization*. 
+. Select organization from the *Organization* list, click *Fix Organization on Mismatch*, then click *Submit*. 
+. Select the checkbox next to the {SmartProxyServer}, click the *Select Actions* list, and select *Assign Location*. 
+. Select location from the *Location* list and select *Fix Location on Mismatch*, then click *Submit*.
+. Click *Administer* > *Organizations* and click on the organization name where you want to see the {SmartProxyServer}.
+. Click *Capsules* on the left-side menu and ensure the {SmartProxyServer} is listed under the *Selected items* list, then click *Submit*.
+. Click *Administer* > *Locations* and click on the location name where you want to see the {SmartProxyServer}.
+. Click *Capsules* on the left-side menu and ensure the {SmartProxyServer} is listed under the *Selected items* list, then click *Submit*.
+. Select the correct organization and location and verify if the {SmartProxyServer} is listed under the following locations by navigating to:
+* Click *Hosts* > *All Hosts*.
+* Click *Infrastructure* > *Capsules*.
+
+
+
+
+
+////
```

Comment:
Why the commented out bits here?

Reply:
Oops, this was meant to be a WIP PR/commit; though didn't mean to push the commented out bits. Fixed, thanks!